code/modules/mob/living/carbon/human/human.dm

```diff
@@ -110,19 +110,21 @@
 		var/perpname = get_face_name(get_id_name(""))
 		if(!HAS_TRAIT(H, TRAIT_SECURITY_HUD) && !HAS_TRAIT(H, TRAIT_MEDICAL_HUD))
 			return
-		var/datum/data/record/R = find_record("name", perpname, GLOB.data_core.general)
+		var/datum/data/record/general_record = find_record("name", perpname, GLOB.data_core.general)
+		var/datum/data/record/med_record = find_record("name", perpname, GLOB.data_core.medical) //SKYRAT EDIT ADDITION BEGIN - EXAMINE RECORDS
+		var/datum/data/record/sec_record = find_record("name", perpname, GLOB.data_core.security)//SKYRAT EDIT ADDITION END
 		if(href_list["photo_front"] || href_list["photo_side"])
-			if(!R)
+			if(!general_record)
```

Comment:
For all instances of you changing R for this, can you add a /// SKYRAT EDIT? Just for us maintainers' sake.

Reply:
Oh and same for R to `sec_records`

Reply:
done!